--- a/include/ntop_defines.h    2019-04-30 21:35:23.614217925 -0400
+++ b/include/ntop_defines.h    2019-04-30 21:36:47.170917483 -0400
@@ -141,9 +141,9 @@
 #define MAX_NUM_VIEW_INTERFACES   8

 #define LIMITED_NUM_INTERFACES    32
-#define LIMITED_NUM_HOST_POOLS    4 /* 3 pools plus the NO_HOST_POOL_ID */
-#define LIMITED_NUM_PROFILES      16
-#define LIMITED_NUM_POOL_MEMBERS  8
+#define LIMITED_NUM_HOST_POOLS    128 /* 4 */ /* 3 pools plus the NO_HOST_POOL_ID */
+#define LIMITED_NUM_PROFILES      128 /* 16 */
+#define LIMITED_NUM_POOL_MEMBERS  256 /* 8 */
 #define UNLIMITED_NUM_INTERFACES  128
 #define UNLIMITED_NUM_HOST_POOLS  128
 #define UNLIMITED_NUM_PROFILES    128
